internal/core/application/types.go

```diff
@@ -256,10 +258,13 @@ func (t *TimeRange) getStartAndEndTime(now time.Time) (startTime time.Time, endT
 			start = time.Date(y, time.January, 1, 0, 0, 0, 0, time.UTC)
 		case All:
 			start = time.Date(StartYear, time.January, 1, 0, 0, 0, 0, time.UTC)
+		case LastYear:
+			lastYear := now.Year() - 1
+			start = time.Date(lastYear, time.January, 1, 0, 0, 0, 0, time.UTC)
+			endTime = time.Date(lastYear, time.December, 31, 23, 59, 59, 0, time.UTC)
```

Comment:
Last year should be the range `[nowMinusOneYear, now]`:
```suggestion
			start = now.AddDate(-1, 0, 0)
```

Reply:
This will subtract one year from current date so as of today value of start would be 2022-01-23 11:33:43.630103 +0100 CET.
We need time range, where start is first minute of first day od prev year.

Reply:
ah! you are actually right! but still, the proposed changes seem wrong to me since we return a time range that covers the whole current year instead of the last one. For example, if I selected LAST_YEAR today, it would return an interval [1/1/23, 31/12/23] that doesn't really make sense since it contains 11 future months.

Reply:
I guess my proposed change is a good starting point to address the issue you correctly pointed out.